### PR TITLE
updated shibboleth proxy in SPID page

### PR DESCRIPTION
**Pull request by peppelinux**
## Description

* chore: older ShibProxy is in EOL, a new shibboleth spid proxy here
* added AgID SPID HelpDesk page

In particular, this shibboleth idp project is hosted to a third-party github repository

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> * chore: older ShibProxy is in EOL, a new shibboleth spid proxy here
- subject may not be empty
- type may not be empty
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 88cc0021f902707657798f12c59bbea0ba75c999
</p>

File: _platforms/en/spid.md [hunk 1]
```diff
@@ -197,6 +209,7 @@ SPID can be integrated in the websites of the Public Administration, but also on
 3. Use a test identity provider such as[spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} or [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} to simulate the authentication flow and verify that your implementation is correct.
 4. Get in touch with other developers at [Slack](https://slack.developers.italia.it/){:target="_blank"}.
 5. Follow the accreditation procedure [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. If you have any further questions or are having problems with onboarding, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```
Comment (bfabio) [inline _platforms/en/spid.md:212]:
```suggestion
6. If you have any further questions or are having problems with the procedure, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```

We call it "accreditation procedure" in the previous point

File: _platforms/it/spid.md [hunk 2]
```diff
@@ -172,6 +172,18 @@ resources:
...
+      icon:
+      desc: Un componente aggiuntivo di Firefox per la visualizzazione dei messaggi SAML inviati tramite il browser durante il Single Sign-On e il Single Logout.
+    - title: Chrome SAML
+      url: https://chrome.google.com/webstore/detail/saml-chrome-panel/paijfdbeoenhembfhkhllainmocckace
+      icon:
+      desc: Un componente aggiuntivo di Chromium per la visualizzazione dei messaggi SAML inviati tramite il browser durante il Single Sign-On e il Single Logout.
+    - title: SAML Developer Tools
+      url: https://www.samltool.com
+      icon:
+      desc: Onelogin SAML2 tools
```
Comment (bfabio) [inline _platforms/it/spid.md:186]: This section on tools is already in `master` for both languages, could you rebase the PR?

File: _platforms/it/spid.md [hunk 3]
```diff
@@ -201,6 +213,7 @@ L'integrazione di SPID è consentita sia per i siti della Pubblica Amministrazio
 3. Usa un Identity Provider di test come [spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} oppure [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} per simulare il flusso di autenticazione e verificare che la tua implementazione sia corretta.
 4. Entra in contatto con gli altri sviluppatori in [Slack](https://slack.developers.italia.it/){:target="_blank"}.
 5. Segui la procedura di accreditamento [descritta nel sito di AGID](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. Se hai ulteriori domande o stai riscontrando problemi con la procedura di accreditamente consulta la pagina di servizio [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add).
```
Comment (bfabio) [inline _platforms/it/spid.md:216]:
```suggestion
6. Se hai ulteriori domande o stai riscontrando problemi con la procedura di accreditamento consulta la pagina di servizio [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add).
```

**Review by bfabio — CHANGES_REQUESTED** [review]

File: _platforms/en/spid.md (hunk 1 above)
Comment (bfabio) [reply _platforms/en/spid.md:212]: Looking at the helpdesk I'm wondering, is that the correct site for someone having troubles when onboarding?

image

I wouldn't know what button to pick.

Comment (peppelinux) [reply _platforms/en/spid.md:212]: Good question ... Anyway that's AgID's help desk page, I'd go further with this mention then come back on this topic in a different way

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> onboarding

fix: accreditation procedure is now "onboarding"
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9960f3794a2f5765f2ac67f998b7f7296d42dcd6
</p>

File: _platforms/en/spid.md (hunk 1 above)
Comment (peppelinux) [reply _platforms/en/spid.md:212]: > We call it "accreditation procedure" in the previous point

I made a commit that normalize this to "onboarding", does it sound good to you?

File: _platforms/it/spid.md (hunk 3 above)
Comment (peppelinux) [reply _platforms/it/spid.md:216]: > We call it "accreditation procedure" in the previous point

I made a commit that normalize this to "onboarding", does it sound good to you?

File: _platforms/it/spid.md (hunk 2 above)
Comment (peppelinux) [reply _platforms/it/spid.md]: done sir!

File: _platforms/en/spid.md [hunk 4]
```diff
@@ -196,7 +196,8 @@ SPID can be integrated in the websites of the Public Administration, but also on
 2. Use and contribute to the open source components available in [Developers Italia](https://github.com/italia?q=spid).
 3. Use a test identity provider such as[spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} or [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} to simulate the authentication flow and verify that your implementation is correct.
 4. Get in touch with other developers at [Slack](https://slack.developers.italia.it/){:target="_blank"}.
-5. Follow the accreditation procedure [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+5. Follow the onboarding [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. If you have any further questions or are having problems with onboarding, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```
Comment (bfabio) [inline _platforms/en/spid.md:200]: I think "onboarding" can't be used as name

"Follow the onboarding procedure"
and
"[...] problems when onboarding"

sound better to me.

**Review by bfabio — APPROVED** [review]
:+1:

File: _platforms/en/spid.md (hunk 4 above)
Comment (peppelinux) [reply _platforms/en/spid.md]: that's right, done!

Comment (peppelinux) [reply _platforms/en/spid.md]: done!

**Comment by peppelinux** [comment]
@bfabio you have the power

**Pull request merged**